.travis.yml

```diff
@@ -13,6 +13,7 @@ env: BUNDLE_PATH=bundle-vendor/bundle
 # will kick in and use `vendor`, ignoring our BUNDLE_PATH
 # declaration in `.bundle/config`
 install:
+- bundle update  
```

Comment:
I don't think updating all gems is necessary.  If you were trying to update the bundler version, it is hardcoded locally in `Gemfile.lock`